docs/resources/Invite.md

```diff
@@ -43,7 +43,8 @@ Represents a code that when used, adds a user to a guild or group DM channel.
     "features": ["NEWS", "DISCOVERABLE"],
     "verification_level": 2,
     "vanity_url_code": null,
-    "nsfw_level": 0
+    "nsfw_level": 0,
+    "premium_subscription_count": 69
```

Comment:
Field is safe to add, but the indention on the fields below `icon` have an extra space or two.

Reply:
Might be blind but i cant see any extra spaces, mind doing a code suggestion?

Reply:
There aren't any extra spaces, it's just how github renders it.
![image](https://user-images.githubusercontent.com/22574513/157513414-c2ddcf1b-84ff-4e38-8d16-af44fd7d58ed.png)
![image](https://user-images.githubusercontent.com/22574513/157513531-c11a6332-30c3-4f01-9d91-598c80d8d694.png)

Reply:
My bad, bamboozled by github ui then.